File: bronze/10101/c++/main.cpp
#include <iostream>

int main()
{
    int a, b, c;
    std::cin >> a >> b >> c;

    int sum = a + b + c;

    if (a == 60 && a == b && a == c && b == c) {
        std::cout << "Equilateral";
    } else if (sum == 180) {
        if ((a == b) || (a == c) || (b == c)) {
            std::cout << "Isosceles";
        } else {
            std::cout << "Scalene";
        }
    } else {
        std::cout << "Error";
    }
    return 0;
}

File: bronze/10101/python/main.py

if (__name__ == "__main__"):
    a = int(input())
    b = int(input())
    c = int(input())

    if (a == 60 and a == b and a == c and b == c):
        print("Equilateral")
    elif ((a+b+c) == 180):
        if (a == b or a == c or b == c):
            print("Isosceles")
        else :
            print("Scalene")
    else :
        print("Error")